.cirrus.yml

```diff
@@ -210,9 +209,12 @@ build_each_commit_task:
 
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     build_each_commit_script:
-        - 'source $SCRIPT_BASE/lib.sh'
-        - 'git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH |& ${TIMESTAMP}'
-        - 'make build-all-new-commits GIT_BASE_BRANCH=origin/$CIRRUS_BASE_BRANCH |& ${TIMESTAMP}'
+        # set -x by default, no need to spew contents of lib.sh
+        - 'source $SCRIPT_BASE/lib.sh &> /dev/null'
+        # Must use $DEST_BRANCH as task runs post-merge
+        # $CIRRUS_BASE_BRANCH is only set when testing a PR!
```

Comment:
Is it worth deleting this one too? Since this is now the only mention of `CIRRUS_BASE_BRANCH` in the entire repo, it really isn't very helpful as a comment except to some expert who knows everything about Cirrus.

And that's probably a good moment for me to make an aside about target audience. It is impossible to know who will be our future maintainers, but IMO it is safer to bet that the next few people touching this file will be Podman experts who have to force themselves to learn Cirrus, not Cirrus experts who are playing with podman. As such, this is probably one of the incredibly rare cases where I'd suggest completely eliminating a comment! Both lines! They only confuse the non-Cirrus-expert.  What I _would_ suggest -- not necessarily for this PR -- is to document the heck out of line 17, where `DEST_BRANCH` is defined, saying something like 'you will almost never need to touch this, but it is imperative to redefine it for maintenance branches and ...etc'.

Reply:
Ohh a COMMIT message...dang I need new glasses, I kept reading COMMENT....yes of course.

So far a few podmen have dabbled toward grokking the Cirrus-CI setup here, and been mildly successful. So I'd say our setup isn't completely unapproachable or "expertise required", but it's also not trivial by any stretch.

I don't really understand about _removing_ the comments, but I'm heavily bias, so I'll take your word for it.  Yes, I do agree `$DEST_BRANCH` needs some explanation.

okay, easy enough fixes.  Thanks again.